### PR TITLE
refactor: fix bad code style

### PR DESCRIPTION
**Pull request by KoNekoD**

### CODE REVIEW

**Review by velenyx — CHANGES_REQUESTED** [review]
Serega лучший чемпион фронтенда 🐳

File: assets/index.html [hunk 1]
```diff
@@ -1,13 +1,90 @@
...
-  </head>
-  <body>
-    <div id="root"></div>
-    <script src="/src/index.tsx" type="module"></script>
-  </body>
+</head>
+<body>
+<div id="root"></div>
+<script src="/src/index.tsx" type="module"></script>
+<script>
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply assets/index.html]: revert 433354f5e7b2cd7d1aac40bfb1ac5f3ca70105ad

File: assets/index.html [hunk 2]
```diff
@@ -1,13 +1,90 @@
...
+                        unusedClasses.push(className);
+                    }
+                });
+            });
+
+            console.log('Неиспользуемые классы:', [...new Set(unusedClasses)]);
+
+        }, 1000 * 5);
+    });
+</script>
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply assets/index.html]: fix 433354f5e7b2cd7d1aac40bfb1ac5f3ca70105ad

File: assets/src/components/monitor/server/parts/body/processes-list/program.tsx [hunk 3]
```diff
@@ -15,25 +15,23 @@ export const Program = ({ group, server }: { group: ApiProcessGroup; server: Api
...
-          </div>
-          <div className='flex flex-grow items-center justify-between'>
-            {/* Костыль чтобы кнопки были на всю ширину */} &nbsp;
+      <div className='flex items-center space-x-2'>
+        <div>
+          <div className='flex cursor-pointer items-center space-x-2 program-title' onClick={() => setIsActive(!isActive)}>
+            {isActive ? <FaChevronUp className='h-4 w-4' /> : <FaChevronDown className='h-4 w-4' />}
             <div>
-              <Buttons server={server} group={group} />
+              <a>{group.name}</a>
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply assets/src/components/monitor/server/parts/body/processes-list/program.tsx:23]: Decorative behavior

File: assets/src/components/monitor/server/parts/header/server-info.tsx [hunk 4]
```diff
@@ -7,13 +7,13 @@ export const ServerInfo = ({ item }: { item: ApiSupervisor }) => {
     return <Fail />;
   }
 
-  const serverTitle = item.server.ip;
-  const serverPort = `:${item.server.port}`;
-  const serverFullTitle = `${serverTitle}${serverPort}`;
+  const ip = item.server.ip;
+  const port = item.server.port;
+  const full = `${ip}:${port}`;
```
[the remark this answers is not in the document]
Comment (KoNekoD) [reply assets/src/components/monitor/server/parts/header/server-info.tsx]: fix 578aa77a4c281a6993296536642f8637af1de83f

**Review by velenyx — APPROVED** [review]
Congratulations 🚀🚀⚡

**Pull request merged**